fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryInsertPlan.java

```diff
@@ -77,7 +77,7 @@ public PipelineOperation getPipelineOperation() {
 
     @Nonnull
     @Override
-    public <M extends Message> CompletableFuture<FDBStoredRecord<M>> saveRecordAsync(@Nonnull final FDBRecordStoreBase<M> store, @Nonnull final M message) {
+    public <M extends Message> CompletableFuture<FDBStoredRecord<M>> saveRecordAsync(@Nonnull final FDBRecordStoreBase<M> store, @Nonnull final M message, final boolean isDryRun) {
         return store.saveRecordAsync(message, FDBRecordStoreBase.RecordExistenceCheck.ERROR_IF_EXISTS);
```

Comment:
Shouldn't this call `dryRunSaveRecordAsync` if `isDryRun` is true? I guess the PR only claims to be about using dry run for updates, but it seems weird that it isn't also implemented for inserts and delete plans